Hymns/Tags/TagStore.swift

```diff
@@ -60,6 +75,21 @@ class TagStoreRealmImpl: TagStore {
             action(favorite)
         }.toNotification()
     }
+
+    // MARK: Realm Query Functions
+
+    /** Can be used either with a value to specificially query for one tag or without the optional to query all tags besides favorites*/
```

Comment:
what do you mean "besides favorites"

Reply:
ohhh I get it

Reply:
It meant not including tags that where favorited. Also, not as relevant anymore since favorites obviously won't be included.